packages/sdk-js/src/util.ts

```diff
@@ -328,3 +328,23 @@ export function loadSDKVersion(): string {
   }
   return version;
 }
+
+export function mergeQueryStrings(oldUrl: string, newUrl: string): string {
```

Comment:
Unit test this function and also test via urlRedirect cases when `persistQueryString` is `true`

Reply:
No need to unit test because it's being tested via cases.json